.clang-format

```diff
@@ -60,4 +60,5 @@ SpaceBeforeAssignmentOperators: true
 
 ContinuationIndentWidth: 4
 SpaceBeforeParens: ControlStatements
+SortIncludes: false
 ...
```

Comment:
:D

---
In reply to: [82900097](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r82900097) [](ancestors = 82900097)